app/controllers/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ForgotPasswordsController < ApplicationController
+  def create
+    user = User.where(email: params[:email]).first
```

Comment:
You should use `find_by` for single record search in Rails 4+. It is more effective than `where(...).first` which was popular in Rails 3.